app/views/publish/courses/index.html.erb

```diff
@@ -2,7 +2,9 @@
 <%= content_for :before_content, render_breadcrumbs(:courses) %>
 
 <h1 class="govuk-heading-l">
-  <span class="govuk-caption-l"><%= @recruitment_cycle.title %></span>
+  <% if @provider.rolled_over? && FeatureService.enabled?(:new_publish_navigation) %>
```

Comment:
Seeing as we've duplicated this check in a few places - could we put this in a helper somewhere?

Reply:
Nice one. Done